picmaker.py

```
@@ -404,7 +404,7 @@ def main():
     group.add_option("-g", "--gamma", dest="gamma",
         action="store", type="float", default=1.,
         help="the gamma value to apply to grayscale of the image returned; "   +
-             "default 1.")
+             "default 1. Values > 1 make grays darker.")
```

Ruff (0.14.7)

Explicitly concatenated string should be implicitly concatenated

Remove redundant '+' operator to implicitly concatenate

(ISC003)